src/pages/auth/components/LogInForm.jsx

```diff
@@ -68,12 +65,14 @@ export const LogInForm = () => {
             isError={errors.password && touched.password}
           />
           <Button
-            style={{ marginTop: '8px' }}
+            buttonTextProps={{ lineHeight: 24, fontSize: 18 }}
+            style={{ marginTop: '15px' }}
             type='submit'
             fullWidth
-            rightIcon={<LoginIcon />}
-            title={t('Log In')}
+            rightIcon={<LoginIcon color={'white'} />}
+            title={isLoading ? t('Loading') : t('Log In')}
```

Comment:
the same, you've already put isLoading prop, why you don't use this logic inside for every buttons?

Reply:
done